weather.py

```diff
@@ -24,19 +25,22 @@ def convert_date(iso_string):
     Returns:
         A date formatted like: Weekday Date Month Year e.g. Tuesday 06 July 2021
     """
-    pass
+    #Answer
+    date = datetime.fromisoformat(iso_string)
+    return(date.strftime("%A %d %B %Y"))
```

Comment:
`return` is a special keyword that returns the value to its right. The code is a little bit easier to read if you leave a space, as return() looks like function e.g.

```suggestion
    return date.strftime("%A %d %B %Y")
```